runtime/lua/vim/uri.lua

```diff
@@ -29,7 +28,12 @@ do
     -- https://tools.ietf.org/html/rfc3986#section-2.2
     rfc3986 = "^A-Za-z0-9%-._~!$&'()*+,;=:@/";
   }
-  local tohex, sbyte = bit.tohex, string.byte
+  local sbyte, tohex = string.byte
+  if jit then
+    tohex = require'bit'.tohex
+  else
+    tohex = function(b) return string.format("%02x", b) end
+  end
```

Comment:
I think this might not be necessary, if people chose to build neovim with another lua than luajit they're supposed to make the bitop package a dependency ( https://github.com/neovim/neovim/issues/11301#issuecomment-546668666 ).

Reply:
Our tests don't seem to agree :P https://travis-ci.org/neovim/neovim/jobs/609437664?utm_medium=notification&utm_source=github_status the travis test doesn't have bitop available.

Reply:
> Our tests don't seem to agree :P https://travis-ci.org/neovim/neovim/jobs/609437664?utm_medium=notification&utm_source=github_status the travis test doesn't have bitop available.

Are you sure? https://travis-ci.org/neovim/neovim/jobs/609437664#L4593

Reply:
what is poorly setup? the test seems to indicate that this code is needed for now, just as claimed.